sandbox/settings.py

```diff
@@ -636,8 +636,8 @@ def post_setup(cls):
                 release=get_release(),
                 integrations=[DjangoIntegration()],
             )
-            with sentry_sdk.configure_scope() as scope:
-                scope.set_extra("application", "backend")
+            scope = sentry_sdk.Scope.get_current_scope()
```

Comment:
We should not use `get_isolation_scope()` ? According to the documentation added in your commit body : `If you want your scope change to affect the whole transaction, use the isolation scope.`

Reply:
Actually, I believe we should not use scope to do that now but contexts.

https://docs.sentry.io/platforms/python/enriching-events/context/#additional-data
IMO, we should do that :
https://github.com/openfun/joanie/blob/758f2e4d05b70ca9ade57033bd35888f323c16a0/src/backend/joanie/settings.py#L620